kafka-ui-react-app/src/components/contexts/ReadOnlyContext.ts

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const initialValue: { isReadOnly: boolean | undefined } = {
+  isReadOnly: undefined,
+};
+const ReadOnlyContext = React.createContext(initialValue);
```

Comment:
It's probably better to call it `ClusterContext` as we store an object there. Another option to use it with only boolean value and call something like this `ClusterReadyOnlyContext`.